pkg/dependencymagnet/doc.go

```diff
@@ -0,0 +1,10 @@
+// +build tools
+
+// go mod won't pull in code that isn't depended upon, but we have some code we don't depend on from code that must be included
+// for our build to work.
+package dependencymagnet
```

Comment:
Is "dependencymagnet" a tool? Or just this process of pinning a dep in doc.go? We recently did this in Hive, curious if there's a more official way.

Reply:
There isn't a more official way.  I copied what KASO did.
https://github.com/openshift/cluster-kube-apiserver-operator/blob/master/pkg/dependencymagnet/doc.go
In upstream kube they do something similar in a different location
https://github.com/kubernetes/kubernetes/blob/master/build/tools.go